python-kubernetes/with-agent/README.md

```diff
@@ -3,6 +3,10 @@
 This is a full example for deploying an application with Rookout using Kubernetes.
 Here we are using a local ETL Agent deployed in the cluster that the SDK will connect to.
 
+Before following this guide we recommend reading the basic [Python + Rookout](https://docs.rookout.com/docs/sdk-setup.html) guide.
```

Comment:
? I updated it a week ago

Reply:
I just noticed that we don't have this preview in this example, though we have it in all the others.